src/main/java/co/grandcircus/WatchYourBackpack/Controllers/MainController.java

```diff
@@ -167,11 +171,13 @@ public ModelAndView startGame(String parkCode, Long id) {
 
 		if (chosenPlayer.equals(null)) {
```

Comment:
Our static analysis engine detects that your pull request includes a change for calling to `equals(null)`. According to the contract of the `equals()` method, this call should always return false.